browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1688,6 +1688,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		if (data.enabled === false) {
 			container.disabled = true;
 			listbox.disabled = true;
+			container.setAttribute('disabled', '');
```

Comment:
why? this is `data.enabled === false` branch

Reply:
Yes, that means it's disabled, that's where attributes are set. Similar thing happens in spinfield, unobutton , etc...

Reply:
ah right, it adds empty but present attr